CHANGELOG.rst

```diff
@@ -4,6 +4,15 @@ Dellemc.Powerstore Change Logs
 
 .. contents:: Topics
 
+v3.7.0
+======
+
+Minor Changes
+-------------
+
+- Host module is enhanced to support check and diff mode.
+- Volume module is enhanced to support check and diff mode.    
```

Comment:
Add bug fix for volume module as well.
"Added support to modify size and host in the same task"

Reply:
done